src/NewRecipe.css

```diff
@@ -0,0 +1,65 @@
+.New {
+  margin-bottom: 50px auto;
+  margin-top: 0px auto;
+  background-color: #282c34;
+  min-height: 100vh;
+  clear: both;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
+
+.New-Body {
+  display: flex;
+  width: 500px;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 25px;
+}
+
+.New-Parameters {
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  margin-top: 15px;
+  color: white;
+}
+
+.New-Submit {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  margin-top: 25px;
+  border-top: solid lightgray;
+  padding: 25px;
+}
+
+.New-InputColor {
+  background-color: rgb(200, 200, 200);
+}
+
+.New-TextColor {
```

Comment:
if these two classes are the same, you can mark them as the same. CSS is normally the time killer when pages load so if it's not 100% necessary you should condense whatever you can.